packages/calcite-components/src/components/meter/readme.md

```diff
@@ -0,0 +1 @@
+# calcite-meter
```

Comment:
Should we add some `usage` files with examples?

Reply:
Yep, added.